localization/english/bookmark/bookmark_583_l_english.yml

```
@@ -1,11 +1,40 @@
+ 
+ It is not only a story of kings and warchiefs, though, for unkown to sides, war is a chess game between sorcerers mastermind and their secret agenda. The human Medivh and the orc Gul'Dan opened the Dark Portal for reasons of their own, and they are not done, yet...
+ 
+ #Will the Horde plunge Azeroth into endless bloodshed ? Will the orcs find a new world to inhabit, or will they pushed back to the doomed Draenor ? What darker schemes await for Azeroth ?#!"
 
 
  #Llane Wrynn
  llane_wrynn_character:0 "King Llane"
- llane_wrynn_character_desc:0 "PH"
+ llane_wrynn_character_desc:0 "A knightly king, Llane was raised in the bond of brotherhood with his close friends knight-champion Anduin Lothar and Medivh, the Guardian. Alongside his friends and advisors, the young king defeated the Trolls who killed his father during the Gurubashi War and proved himself true to the legacy of Wrynn dynasty by making his kingdom the mightiest of all humanity. With the invasion from Draenor incoming, Llane now has to face a threat more terrible than any of his ancestors did.
+ #The war to come may bring doom to Llane's entire people and legacy, but were he to succeed, he would be remembered as the greatest of all kings. Can his friends help him repel the incoming darkness ?#!
+ "
+
+ #Medivh
+ medivh_character:0 "Guardian Medivh"
+ medivh_character_desc:0 "Son of the court conjurer of Stormwind and of the Guardian of Tirisfal, Medivh is the greatest scion of a spellcaster lineage as well as close friend to king Llane whom he helped saving Stormwind during the Gurubashi War. Having succeeded his mother as Guardian, Medivh might be the most powerfull mage in the world, tasked to safeguard Azeroth from the most dire threats ; however, he has not been sighted out of his own lordship of Karazhan for some times, his reclusion could be due to his excentric personnality or some wizard's pet project ?
+ #Will Medivh be able to save Stormwind once again ? Does he have another plan ? What fate awaits the tower of Karazhan in the middle of the war to come ?#!
+  "
+
+ #Blackhand
+ blackhand_character:0 "Warchief Blackhand"
+ blackhand_character_desc:0 "A force of nature whose military prowess has earned him the nickname of "destroyer", Blackhand is chieftain of the Blackrock clan, the most powerfull and populous of the Horde. Those two reasons led him to be chosen by his fellow chieftains and the influent warlock Gul'Dan to be warchief of the Horde and military overlord of the united orcs. With Draenor conquered but the Horde starving of both combat and ressources, the warchief is proud to lead his warriors into the conquest of Azeroth and secure both new lands to rule and his position as the one true leader of the orcs despite the will of idealistic chieftains, ambitious commanders and scheming warlocks...
```

Shortening some sentence and minor adjustment.
```suggestion
  blackhand_character_desc:0 "A force of nature, whose military prowess has earned him the nickname of "the destroyer". Blackhand is the chieftain of the Blackrock clan, the most powerful and populous clan of the Horde. Those two reasons led him to be chosen by his fellow chieftains and the influential warlock Gul'Dan to be warchief of the Horde and military overlord of the united orc clans. With Draenor conquered but the Horde starving of both combat and resources, the warchief is proud to lead his warriors into the conquest of Azeroth. The conquest will secure both new lands to rule and his position as the one true leader of the orcs despite the will of idealistic chieftains, ambitious commanders, and scheming warlocks...
```